diff --git a/node_modules/react-flipcard/lib/components/FlipCard.js b/node_modules/react-flipcard/lib/components/FlipCard.js
index 4c816ba..7a0b005 100644
--- a/node_modules/react-flipcard/lib/components/FlipCard.js
+++ b/node_modules/react-flipcard/lib/components/FlipCard.js
@@ -23,19 +23,23 @@ var _helpersContains2 = _interopRequireDefault(_helpersContains);
 var _helpersInjectStyle = require('../helpers/injectStyle');
 
 var _helpersInjectStyle2 = _interopRequireDefault(_helpersInjectStyle);
+// 에러 해결을 위해 추가한 옵션
+var PropTypes = require('prop-types');
+var createReactClass = require('create-react-class');
 
 // Auto inject the styles (will only be done once)
 (0, _helpersInjectStyle2['default'])();
 
-exports['default'] = _react2['default'].createClass({
+exports['default'] = createReactClass({
   displayName: 'ReactFlipCard',
 
   propTypes: {
-    type: _react.PropTypes.string,
-    flipped: _react.PropTypes.bool,
-    disabled: _react.PropTypes.bool,
-    onFlip: _react.PropTypes.func,
-    onKeyDown: _react.PropTypes.func,
+    // 에러 해결을 위해 _react.PropTypes를 PropTypes로 변경
+    type: PropTypes.string,
+    flipped: PropTypes.bool,
+    disabled: PropTypes.bool,
+    onFlip: PropTypes.func,
+    onKeyDown: PropTypes.func,
     children: function children(props, propName, componentName) {
       var prop = props[propName];
 
